src/pyhf_benchmark/plot.py

```diff
@@ -98,6 +127,20 @@ def subplot(y_label, column, output, directory, filename):
 
 
 def subplot_comb(y_label, column, outputs, backends, directory, filename):
+    """
+    Plot each metrics for the running results if multiple lines are plotted in one graph.
+
+    Args:
+        y_label: Y axis Name
+        column: Value for Y axis
+        outputs: CPU and GPU work status contents
+        backends: Backends information
+        directory: Directory for plot file
+        filename: Filename for plot file
+
+    Returns:
+
```

Comment:
As I mentioned before, probably makes more sense to have the function accept an axis object and then return an axis object then try to wrap all of it, but that can be done for a later PR. At the moment this doesn't return anything so it should get fixed.